package.json

```diff
@@ -17,6 +17,10 @@
   },
   "dependencies": {
     "@snowdog/alpaca-components": "^0.3.0-alpha.5",
+    "@storybook/addon-backgrounds": "^5.0.11",
+    "node-sass": "^4.11.0",
+    "sass-loader": "^7.1.0",
+    "vue": "^2.6.8",
```

Comment:
This one is a peer dep and should remain like this to ensure Vue instance is shared with the project